src/reducers/chord_sheet_reducer.js

```diff
@@ -0,0 +1,115 @@
+import exampleChordProSheet from '../utils/example_chord_pro_sheet';
+import { decompress } from '../utils/string_compression';
+import getQueryParam from '../utils/get_query_param';
```

Comment:
This is one of the things you should not do in redux! consider using middlewares